topic_folders/policies/incident-response.md

```diff
@@ -45,5 +45,25 @@ Individuals reported often get upset, defensive, or deny the report. Allow them
 - It is not your job to reassure or forgive them.
 - Do not allow the reported person to make an apology to the reporter or impacted person. Often an apology centers the reported person’s feelings and not the person who was impacted. You may accept their apology and offer to pass it on, but you’re not required to if you think it would negatively impact the reporter.
 
+#### Incidents involving Core Team members
+
+* If there are CoC reports either about or from (or in any way involving) Core
+  Team members who are Community Initiatives (CI) employees, and those reports
+  have the potential to rise to the level of harassment as defined by CI's
+  handbook, CI HR must be made aware immediately.
+* If a PEO employee (non-U.S. based Core Team members) is involved, we will
+  need to bring the PEO for country-specific advice.
+* In cases involving CI employees that are definitely not harassment, it is ok
+  to handle through CoCc without involving CI. Examples of types of things that
```

Comment:
Not sure adding two small examples here helps or rather confuses the person that needs to decide whether something is harassment or not.